measure.rb

```diff
@@ -1932,6 +1953,26 @@ def self.add_hot_water_and_appliances(runner, model, building, weather, spaces)
     return true
   end
 
+  def self.get_desuperheatercoil(hvac_map, relatedhvac, wh_id)
+    # search for the related cooling coil object for desuperheater
+
+    # Supported cooling coil options
+    clg_coil_supported = [OpenStudio::Model::CoilCoolingDXSingleSpeed, OpenStudio::Model::CoilCoolingDXMultiSpeed, OpenStudio::Model::CoilCoolingWaterToAirHeatPumpEquationFit]
+    if hvac_map.keys.include? relatedhvac
+      target_hvac = hvac_map[relatedhvac]
+      target_hvac.each do |comp|
```

Comment:
Seems like there's no point to create the `target_hvac` variable. Just go with `hvac_map[relatedhvac].each do |comp|`.